examples/basics/lists/variable-list/variable-list.go

```diff
@@ -0,0 +1,71 @@
+package main
```

Comment:
We should add this example to the README as well: https://github.com/qlik-oss/enigma-go/blob/master/examples/basics/lists/README.md